### DISCUSSION TITLE
How to configure multiple controllers with same region name

### DISCUSSION THREAD

**Discussion by VickyXie777** [naver/ngrinder#829] [Q&A]
Hi, we want to deploy multiple controllers with same region name to perform load balance by using L7 load balancer at the controller layer when perftest requests come in. But when i boot the application, I got the exception shown below:

![image](https://user-images.githubusercontent.com/46780837/133058254-5efeab60-d67c-4a6d-8b2e-33158b6190cb.png)

Could you guys provide a work around solution? 
@junoyoon

**Comment by donggyu04** [comment]
@VickyXie777 

multiple controllers with the same region are not supported.
why do you want to load balance between controllers?
If you want to add a load balancer in front of controllers you must run with clutering mode, check below

https://github.com/naver/ngrinder/wiki/Advanced-Clustering-Guide

**Reply by VickyXie777** [reply]
Cause we met the situation once, when the controller is distributing large files under security mode, other perftests are all pending, that's not what we expect, so if we want to start some perftests at the same time with load from the same region, multiple controllers can handle such issue, I think. (for our infrastructure, subregion is not necessary)

And ofcourse we already added load balancer, thanks a lot.

**Comment by donggyu04** [comment]
@VickyXie777 

If your distributing file is too large. you can manually upload the files before testing and refer to them as an absolute path in the script.
or if files are libraries, you can add the path to `agent.jvm.classpath` option that is located in `agent.conf` file.

> `agent.conf` is located in `{USER_HOME}/.ngrinder_agent`
